src/core/Radio/index.tsx

```diff
@@ -37,6 +37,7 @@ const RadioButton = (props: RadioProps): JSX.Element => {
 
   return (
     <StyledRadioButton
+      data-testid="radioButton"
```

Comment:
I think you should not add a `data-testid` here, in the main component code! 
You can add it to the story function and use it! 
You can take a look at the `InputSlider` component for an example. In the `index.stories.tsx` you'll find a `TestDemo` function which returns a JSX element with a `data-testid`.